src/test/scenarios/managed-network/input/managedNetwork.json

```diff
@@ -89,7 +89,8 @@
             "schema": {
               "$ref": "#/definitions/ManagedNetwork"
             },
-            "description": "Parameters supplied to the create/update a Managed Network Resource"
+            "description": "Parameters supplied to the create/update a Managed Network Resource",
+            "x-ms-client-flatten": true
```

Comment:
can we add "x-ms-client-flatten" to managed network in Azure REST API specification?
I am worried it may cause breaking change in the SDK, so I am not sure about it